courses/free-courses-en.md

```diff
@@ -382,6 +382,7 @@
 
 #### SQL
 
+* [CockroachDB University](https://university.cockroachlabs.com) - Cockroach Labs, Inc. *(email address required)*
```

Comment:
let's try to avoid generic links, please add link(s) to the courses you want

Reply:
Ok, I have removed the generic link and have added two of the courses that I would like to add to the SQL list.